Editor/Window/RegisterComputeInput.cs

```diff
@@ -88,7 +88,7 @@ private async void OnRegisterComputeButtonClicked()
             {
                 var previousComputeName = _stateManager.ComputeName;
                 var registerResponse = await _computeManager.RegisterFleetCompute(_computeName,
-                    _stateManager.AnywhereFleetId, _location, _ipAddress);
```

Comment:
`_location` doesn't appear to be used, please remove it

Reply:
Removed